### PR TITLE
Branch Development of Porfolio

### PR DESCRIPTION
**Pull request by ivanmvh**
- Update Resources
- Update  README.md
- Update index.html
- Update styles.css
- Many commits caused for linters ckecks (spaces)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -92,3 +93,129 @@ button:active {
...
+
+  /* padding-top: 70px;
+  padding-bottom: 70px; */
+}
+
+.tit-projects {
+  margin-left: 40px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
```
Comment (franclobo) [inline css/styles.css:114]: - [x] Please, 🙏🏽 we could add font size to be similar to the Figma template.

- Figma design

image

- Your design

image

File: index.html [hunk 2]
```diff
@@ -10,28 +10,109 @@
...
-   <button>Start collaboration</button>
+      <button>Start collaboration</button>
 
-   <footer>
-       <img class="img2" src="resources/image_geometry_2.svg" alt="Mosaic bottom">
-   </footer>
-</section>     
+      <footer>
+         <img class="img2" src="resources/image_geometry_2.svg" alt="Mosaic bottom">
+      </footer>
```
Comment (franclobo) [inline index.html:31]: - [x] kindly for the best practices for HTML there is only one footer at the end of the web page (contact form), we could use the `section` tag instead. 😃 
- [ ] Please 🙏🏽 adjust the image to fill the space like in the Figma template.

image

File: index.html [hunk 3]
```diff
@@ -10,28 +10,109 @@
...
+
+   <section class="sec-works">
+      <article class="project">
+         <img src="resources/P1-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>               </div>
```
Comment (franclobo) [inline index.html:43]: - [ ] At the moment we don't need `Lorem Ipsum` we will use it later for the pop-up window. Kindly use the title `Project name goes here` instead. 👇🏽 

image

File: index.html [hunk 4]
```diff
@@ -10,28 +10,109 @@
...
+      </article>
+ 
+      <article class="project">
+         <img src="resources/P3-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>
```
Comment (franclobo) [inline index.html:78]: - [ ] At the moment we don't need `Lorem Ipsum` we will use it later for the pop-up window. Kindly use the title `Project name goes here` instead. 👇🏽 

image

File: index.html [hunk 5]
```diff
@@ -10,28 +10,109 @@
...
+         </div>            
+      </article>
+      <article class="project">
+         <img src="resources/P4-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>
```
Comment (franclobo) [inline index.html:95]: - [ ] At the moment we don't need `Lorem Ipsum` we will use it later for the pop-up window. Kindly use the title `Project name goes here` instead. 👇🏽 

image

File: index.html [hunk 6]
```diff
@@ -10,28 +10,109 @@
...
+   <section class="about-me">
+      <h1 class="title">About<br>Me</h1>
+      <p class="parrafo"> 
+         Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+         sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+         nisi ut aliquip ea eiusmod
+      </p>
+      <button class="about-button">Get my CV -&gt; </button>
+   </section>
```
Comment (franclobo) [inline index.html:115]: - [ ] Work section and About me section is incomplete, in the Figma template in the Work section there are six projects, and Languages, Frameworks, and skills are part of the About me section. Kindly complete those sections. ☝🏽 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: css/styles.css [hunk 7]
```diff
@@ -92,3 +93,129 @@ button:active {
...
+  color: #fff;
+}
+
+.menu-hamburg {
+  color: #000;
+  width: 1fr;
+  margin-right: 20px;
+}
+
+.about-myself {
```
Comment (franclobo) [inline css/styles.css:198]: - [ ] May you mean `.about-me`? I couldn't find the class in the HTML file. Please 🙏🏽  apply the requirements for this section:

> Make sure the content of the "About myself" section is aligned with flexbox.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @ivanmvh, 

Good job so far!
You did it well, congratulations!!! 🤩 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Professional README file 💯 
- Correct Github flow ✔️ 
- Descriptive PR 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

- [ ] Complete the work and about sections with the advice below. 👇🏽 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @ivanmvh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

5SW

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**